app/components/shared/Panel/index.js

```diff
@@ -39,9 +40,9 @@ Panel.IntroWithButton = IntroWithButton;
 Panel.Row = Row;
 Panel.RowActions = RowActions;
 Panel.RowLink = RowLink;
+Panel.Header = Header;
 
 const SIMPLE_COMPONENTS = {
-  Header: 'bg-silver py2 px3 semi-bold',
```

Comment:
I figured it was inevitable due to the combination of linter rules not allowing multiple components in a file, and the general inflexibility of this type of code: 

```
const SIMPLE_COMPONENTS = {
  Description: 'dark-gray mt1',
  Icon: 'flex items-center',
  Menu: 'flex items-center'
};

Object.keys(SIMPLE_COMPONENTS).forEach((componentName) => {
```